### PR TITLE
#77 Implement cost and return forms

### PR DESCRIPTION
**Pull request by chuckgreenman**
fixes: #77 
During our meeting with the preservation lab we discovered I hadn't implemented the cost and return tab from the access database.

While writing up the PR I noticed that the currency holding fields aren't formatting properly, planning to look into that.

### CODE REVIEW

**Comment by chuckgreenman** [comment]
I think this is ready to rock and roll now.

**Comment by crowesn** [comment]
@chuckgreenman hey chuck this has at least one conflict. When I do a rebase over qa on local it wants more than just the db/schema.rb

Can you take a look and update the branch?

**Comment by crowesn** [comment]
@hortongn this should be ready for review.

**Comment by hortongn** [comment]
@crowesn Chuck's original writeup for this PR says "While writing up the PR I noticed that the currency holding fields aren't formatting properly, planning to look into that."

Is that still a problem that still needs resolved?  If so, can you write up an issue for it?  I think currency fields look fine, but I'm not sure what Chuck meant.

File: app/helpers/conservation_records_helper.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # frozen_string_literal: true
 
 module ConservationRecordsHelper
+  def date_returned(record)
+    return if record.cost_return_report.blank?
+
+    returned_date = record.cost_return_report.returned_to_origin
+    return returned_date.to_date if returned_date.present?
```
Comment (hortongn) [inline app/helpers/conservation_records_helper.rb:8]: I'd like to see a test that covers lines 7-8 when the cost_return_report exists.

**Pull request merged**